kong/constants.lua

```diff
@@ -36,6 +36,7 @@ local plugins = {
   "azure-functions",
   "zipkin",
   "opentelemetry",
+  "ai-proxy",
```

Comment:
Probably, we should consider hybrid mode compatibility check?

Reply:
It works hybrid for me, and also Konnect for Rick.

Is there some automated check that needs to be added to make sure that it works over hybrid?

Reply:
_Jack note: done_